homeassistant/components/emoncms/manifest.json

```diff
@@ -1,7 +1,10 @@
 {
   "domain": "emoncms",
   "name": "Emoncms",
-  "codeowners": ["@borpin"],
+  "codeowners": ["@borpin", "@arnyminerz"],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/emoncms",
-  "iot_class": "local_polling"
+  "integration_type": "hub",
+  "iot_class": "cloud_polling",
```

Comment:
That is only relevant if using [emoncms.org](https://www.emoncms.org).
Majority of users interface with a local instance, it does not require internet access.